CHANGES.md

```diff
@@ -1,3 +1,11 @@
+## 0.3.1
```

Comment:
generally, we shouldn't actually define a version until we're doing the release (so it should just say e.g. `## 0.X.X` -- it's clearer that it isn't released yet so that when someone else goes here they know not to create e.g. 0.3.2.  In this case it's fine, since we're prepping for the release (though you may want to include a placeholder marker for the next set of changes above this.